src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/isothermalHystInjection/Example.rst

```diff
@@ -0,0 +1,342 @@
+
+.. note::
+    comment on NullSolid
+
+The relative permeability model is input through tables thanks to **TableRelativePermeability** block.
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_drainageOnly_iterative_base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_FLUID -->
+    :end-before: <!-- SPHINX_FLUID_END -->
+
+As this benchmark is testing the sensitivity of the plume dynamics to the relative permeability hysteresis modeling,
+in commented block the **TableRelativePermeabilityHysteresis** block sets up bounding curves for imbibition and drainage
+under `imbibitionNonWettingRelPermTableName`, `imbibitionWettingRelPermTableName` and, `drainageWettingNonWettingRelPermTableNames`
+compared to the `wettingNonWettingRelPermTableNames` label of the drainage only **TableRelativePermeability** blocks.
+Those link to **TableFunction** blocks in **Functions**, which define sample points for piecewise linear interpolation.
+This feature is used and explained in more details in the following section dedicated to *Initial and Boundary conditions*.
+
+See,
+
+.. code-block:: console
+
+  ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_hysteresis_iterative_base.xml
+
+for the input base with hysteresis set up.
+
+.. note::
+    `wettingNonWettingRelPermTableNames` in **TableRelativePermeability** and `drainageWettingNonWettingRelPermTableNames` in
+    **TableRelativePermeabilityHysteresis** usage are identical.
+
+
+Capillary pressure is also tabulated and defined in **TableCapillaryPressure**. No hysteresis is modeled yet on the capillary pressure.
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_drainageOnly_iterative_base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_FLUID -->
+    :end-before: <!-- SPHINX_FLUID_END -->
+
+-----------------------------------------------------------
+Initial and boundary conditions
+-----------------------------------------------------------
+
+The domain is initially saturated with brine with a hydrostatic pressure field.
+This is specified using the **HydrostaticEquilibrium** XML tag in the **FieldSpecifications** block.
+The datum pressure and elevation used below are defined in
+`(Class et al., 2009) <https://link.springer.com/article/10.1007/s10596-009-9146-x>`__).
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_drainageOnly_iterative_base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_HYDROSTATIC -->
+    :end-before: <!-- SPHINX_HYDROSTATIC_END -->
+
+In the **Functions** block, the **TableFunction** s named ``initOilCompFracTable`` and ``initWaterCompFracTable``
+define the brine-saturated initial state, while the **TableFunction** named ``initTempTable`` defines the
+homogeneous temperature field (temperature is not used in this benchmark).   
```

Comment:
Maybe we should remove the part about temperature, because here the temperature is not exactly homogeneous. With the geothermal gradient that we are using, there should be at least 1.5 K of temperature difference between the bottom and the top of the model. It is not much, but still